build-test.cmd

```diff
@@ -344,46 +351,36 @@ if defined __BuildAgainstPackagesArg (
   )
 )
 
-echo %__MsgPrefix%Creating test wrappers...
-
-set RuntimeIdArg=
-set TargetsWindowsArg=
-
-if defined __RuntimeId (
-    set RuntimeIdArg=-RuntimeID="%__RuntimeId%"
-)
-
-if "%__TargetsWindows%"=="1" (
-    set TargetsWindowsArg=-TargetsWindows=true
-) else if "%__TargetsWindows%"=="0" (
-    set TargetsWindowsArg=-TargetsWindows=false
-)
+echo %__MsgPrefix%Creating test overlay...
 
```

Comment:
Looks like this deserves a big header comment:
```
REM =========================================================================================
REM ===
REM === Create the test overlay
REM ===
REM =========================================================================================
```

Reply:
I see it is part of a bigger section. Hmmm... given the new control flow, though, it looks like it's going to be very easy to get confused.

---
In reply to: [198341655](https://github.com/dotnet/coreclr/pull/18365#discussion_r198341655) [](ancestors = 198341655)